tgtg_scanner/notifiers/smtp.py

```diff
@@ -79,29 +96,48 @@ def _stay_connected(self) -> None:
         if status != 250:
             self._connect()
 
-    def _send_mail(self, subject: str, html: str) -> None:
+    def _send_mail(self, subject: str, html: str, item_id: int) -> None:
         """Sends mail with html body"""
         if self.server is None:
             self._connect()
         if self.sender is None or self.recipients is None or self.server is None:
             raise SMTPConfigurationError()
         message = MIMEMultipart("alternative")
         message["From"] = self.sender
-        message["To"] = ", ".join(self.recipients)
+
+        # Contains either the main recipient(s) or recipient(s) that should be
+        # notified for the specific item. First, initalize with main recipient(s)
+        recipients = self.recipients
+
+        # Determine recipient(s) based on item_id
+        if self.item_recipients:
```

Comment:
This is unnecessary as `self.item_recipients` should always be a dict at this point.